src/dbCache.js

```diff
@@ -45,7 +45,7 @@ export default (db) => {
         const result = await promiseQuery(insertImage, vars);
         return result.rowCount === 1;
       } catch (e) {
-        log('error', e.stack);
+        log('error', JSON.stringify(e));
```

Comment:
Small thing: most errors serialize to `{}` as JSON, and some might actually contain circular references. I would either use `e.toString()` or `util.inspect(e)` instead (the later being the most robust)